<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QMainWindow" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_isim">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>185</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>Kullanıcı Adı</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>305</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sifre">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>345</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>Şifre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telno">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>265</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>225</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBack">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(170, 170, 0);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(170, 170, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
    </property>
    <property name="text">
     <string>Zaten Üye misin?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>115</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>TelefonNumarası</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>İsim Soyisim</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_plaka">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>385</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKayit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{

	border: 1px solid #111111;
	border-radius: 5px;
	border-color: rgb(170, 170, 0);
	border-width: 2px;
	margin-left: 4px;
	padding: 5px;
	color:  rgb(0, 0, 0);
	background-color:rgb(170, 170, 255);
	font: 700 9pt &quot;Segoe UI&quot;;



}</string>
    </property>
    <property name="text">
     <string>Kayit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>Plaka</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-weight: bold;
color:rgb(43, 0, 108);
}</string>
    </property>
    <property name="text">
     <string>E-mail adresi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>lineEdit_isim</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>lineEdit_sifre</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_telno</zorder>
   <zorder>lineEdit_email</zorder>
   <zorder>pushButtonBack</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit_plaka</zorder>
   <zorder>pushButtonKayit</zorder>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
